index.md

```diff
@@ -1453,14 +1448,415 @@ def check(condition, message):
 
     Note that it is **strongly recommended** that you [**always use four spaces per indentation level**][PEP8: Indentation] throughout your code.
 
-2.  *IndentationError: unexpected indent*
+5.  *IndentationError: unexpected indent*
 
     In Python, the only time you would increase the indentation level of your code is to define a new code block after a [compound statement][Compound statements] such as `for`, `if`, `def`, or `class`.
 
     ~~~~ {include="unexpected_indent"}
     ~~~~
 
 
+## New errors {#new}
+
+### Consider iterating dictionary (C0201) {#C0201}
+
+It is more *pythonic* to iterate through a dictionary directly, without calling the `.keys` method.
+
+~~~~ {include="C0201_consider_iterating_dictionary"}
+~~~~
+
+Corrected version:
+
+```python
+for item in menu:
+    print("My store sells {}.".format(item))
+```
+
+### Superfluous parens (C0325) {#C0325}
+
+This error occurs when a keyword, such as `if` or `for`, is followed by a single item enclosed in parentheses. In such a case, parentheses are not necessary.
+
+~~~~ {include="C0325_superfluous_parens"}
+~~~~
+
+Corrected version:
+
+```python
+if 'anchovies' in pizza_toppings:  # Error on this line
+    print("Awesome!")
+```
+
+### Literal comparison (R0123) {#R0123}
+
+This error occurs when we use the identity operator `is` to compare Python literals. Whether or not two literals representing the same value (e.g. two identical strings) have the same identity can vary, depending on the way the code is being executed, the code that has ran previously, and the version and implementation of the Python interpreter. For example, each of the following assertions pass if the lines are evaluated together from a Python file, but `assert num is 257` and `assert chars is 'this string fails'` fail if the lines are entered into a Python interpreter one-by-one.
+
+~~~~ {include="R0123_literal_comparison"}
+~~~~
+
+To prevent the confusion, it is advisable to use the equality operator `==` when comparing objects with Python literals.
+
+```python
+num = 256
+assert num == 256
+
+num = 257
+assert num == 257  # Assertion fails if typed into a Python interpreter
```

Comment:
This comment and the one below are weird. Doesn't `num is 257` fail, while `num == 257` succeed?

Reply:
Yes, sorry. I copy-pasted and forgot to remove the comments.